docs/io/ascii/index.rst

```diff
@@ -34,9 +38,14 @@ section on `Supported formats`_ contains the full list.
 The strength of `astropy.io.ascii` is the support for astronomy-specific
 formats (often with metadata) and specialized data types such as
 :ref:`SkyCoord <astropy-coordinates-high-level>`, :ref:`Time
-<astropy-time>`, and :ref:`Quantity <quantity>`. For reading or writing large
-data tables in a generic format such as CSV, using the :ref:`Table - Pandas
-interface <pandas>` is an option to consider.
+<astropy-time>`, and :ref:`Quantity <quantity>`.
+
+.. note::
+
+   For reading large CSV files, the Table :ref:`PyArrow <table_io_pyarrow_csv>` interface
+   is an option to consider since it can be up to 15 times faster than other readers. For
```

Comment:
Just "is recommended"? If not that, either change the wording here or in the second line, to avoid repeating "an option to consider"

Reply:
This second note was actually orphaned and I had intended to cut it. It's done now, with the Pandas-for-writing moved up as well.

Reply:
New note (with the 2nd one removed):
<img width="762" alt="image" src="https://github.com/user-attachments/assets/eacf411d-83a6-4db0-9c2a-73f698931f35" />